docs: describe your change... (#35483)
angular/angular: aio/content/guide/file-structure.md

| APP SUPPORT FILES    | PURPOSE |
| :--------------------- | :------------------------------------------|
| `app/`                 | Contains the component files in which your application logic and data are defined. See details [below](#app-src). |

> | `assets/`              | Contains image and other asset files to be copied as-it is when you build your application. |

Reviewer: How about this?
```suggestion
| `assets/`              | Contains image and other asset files to be copied as they are when you build your application. |
```
